.changeset/orange-seas-tie.md

```diff
@@ -0,0 +1,6 @@
+---
+'@eth-optimism/sdk': major
+'@eth-optimism/contracts-bedrock': minor
```

Comment:
nit: this is definitely a major bump here too

Reply:
This would be the first major release of the `contracts-bedrock` package. There's precedence for API breaking releases that did not warrant a major bump of the package in the past. With this being such a small change + the contracts being pre-prod, this doesn't feel like the right PR to do this in imo. cc @tynes

Reply:
This is a major release of the sdk and a minor release of the contracts which should be fine?

Reply:
We should do the major release of the contracts with the code freeze

Reply:
Yeah, that is compliant with the [README's Versioning section](https://github.com/ethereum-optimism/optimism/tree/develop/packages/contracts-bedrock#versioning).